dnf-behave-tests/features/repo-priorities.feature

```diff
@@ -19,6 +19,9 @@ Scenario: Install an RPM from the highest-priority repository
     And Transaction is following
         | Action        | Package                                   |
         | install       | flac-0:1.3.3-2.fc29.x86_64                |
+   When I execute dnf with args "upgrade"
+   Then the exit code is 0
+    And Transaction is empty
```

Comment:
I don't think we should be testing obsoletes in the `repo-priorities.feature` but If it is somehow related to repo priorities please make a separate scenario.

Reply:
I am sorry, I put the change into the same commit. The code only extend the original test of priority. It ensures that after installing the latest package from priority repo, it did not upgrade it to package from repo that has higher priority value. I realized in the code that it is really easy to break the behaviour. Shall I split it in another commit?